traitsui/editors/tabular_editor.py

```diff
@@ -103,8 +103,13 @@ class TabularEditor(BasicEditorFactory):
     #: trigger a scroll-to command. The data is an integer giving the column.
     scroll_to_column = Str()
 
-    #: Controls behavior of scroll to row
-    scroll_to_row_hint = Enum("center", "top", "bottom", "visible")
+    #: Deprecated: Controls behavior of scroll to row and scroll to column
+    scroll_to_row_hint = Enum("visible", "center", "top", "bottom")
+
+    #: (effectively the same as scroll_to_row_hint, but more clearly named)
+    #: Controls behavior of scroll to row and scroll to column
+    #: defaults to None in which case scroll_to_row_hint is used
+    scroll_to_position_hint = Enum(None, "visible", "center", "top", "bottom")
```

Comment:
like what we discussed - let's make `scroll_to_row_hint` a property where the setter and getter simply set and return `scroll_to_position_hint` - while raising a deprecation warning in the setter and getter.

Reply:
ah yes I had completely forgotten to update this PR!